compendium/test_bundles/TestInterfaces/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(PROJECT_TARGET usTestInterfaces)
+
+project(TestInterfaces VERSION 1.0.0 LANGUAGES CXX)
+
+add_library(usTestInterfaces
+  src/Interfaces.cpp
+  )
+
+configure_file(
+  ${CppMicroServices_SOURCE_DIR}/cmake/Export.h.in
+  ${CMAKE_CURRENT_BINARY_DIR}/include/TestInterfaces/${PROJECT_NAME}Export.h
+  )
+
+include_directories(${CppMicroServices_SOURCE_DIR}/framework/include
```

Comment:
I don't believe both ```target_include_directories``` and ```include_directories``` are needed. They do the same thing just at different scopes (i.e. target vs. directory scope). 

I suggest removing ```include_directories```

Reply:
Done.